### PR TITLE
Speed up BRCTG test to avoid S422 time limit exception

### PR DESCRIPTION
**Pull request by dsh33782**
I added following code inside BRCTG loop to speed it up to prevent BUILD S422 abend due to time limit
    LTR R3,R3            IS LOW HALF ZERO
    BZ  SKIP_RESET   YES, SKIP RESET
    LA  R3,1               NO, RESET LOW R3 TO 1 FOR LAST ITERATION
SKIP_RESET EQU *

Note I also added extra WTO so you can see resets

### CODE REVIEW

File: rt/mlc/TBRCTX.MLC [hunk 1]
```diff
@@ -60,15 +61,26 @@ BRCTH_1  DS    0H
 *
          SGR   R2,R2               Count iterations
          SGR   R3,R3               Initialize all 64 bits
-         LG      R3,=FL8'3'    Loop count  #370 DSH REP IIHF WITH LG
+         IIHF    R3,3                 Loop count
 BRCTG_1  DS    0H
          AGHI  R2,1                Count iterations (64-bit counter)
+*
+* CODE ADDED BY DSH TO SPEED UP BRCTG TEST
```
Comment (jyganci) [inline rt/mlc/TBRCTX.MLC:68]: Suggestion only. Change "SPEED UP" to "SHORTEN".

**Review by jyganci — APPROVED** [review]
Looks okay.

File: rt/mlc/TBRCTX.MLC [hunk 2]
```diff
@@ -60,15 +61,26 @@ BRCTH_1  DS    0H
...
+*
+         LTR   R3,R3               IS LOW R3 ZERO   
+         BZ    SKIP_RESET          YES, SKIP RESET AND PRINT
+         LA    R3,1                NO, RESET TO 1 FOR LAST ITERATION
+         LA    R1,LblBRCTG         Label for message
+         BRAS  R10,ShowMsg         Show message
```
Comment (jyganci) [inline rt/mlc/TBRCTX.MLC:74]: While misleading -- the message starts with "After BRCTG" -- it does show the decrement of the high half of GR3. Leave it in, I guess.

File: rt/mlc/TBRCTX.MLC [hunk 3]
```diff
@@ -60,15 +61,26 @@ BRCTH_1  DS    0H
...
+SKIP_RESET EQU *
+*
          BRCTG R3,BRCTG_1          Loop
 *
+PRINT3 EQU *
```
Comment (jyganci) [inline rt/mlc/TBRCTX.MLC:79]: PRINT3 not referenced. Line can be deleted.

File: rt/mlc/TBRCTX.MLC (hunk 2 above)
Comment (dsh33782) [reply rt/mlc/TBRCTX.MLC:74]: ok

File: rt/mlc/TBRCTX.MLC (hunk 3 above)
Comment (dsh33782) [reply rt/mlc/TBRCTX.MLC:79]: ok

**Pull request merged**